backend/models.py

```diff
@@ -41,9 +41,9 @@ class Participant(db.Model):
 
     __tablename__ = "participant"
     id = Column(Integer, autoincrement=True, primary_key=True)
-    first_name = Column(String(50))
-    last_name = Column(String(50))
-    email = Column(String(200))
+    first_name = Column(String(50), nullable=False)
+    last_name = Column(String(50), nullable=False)
+    email = Column(String(200), unique=True, nullable=False)
     github = Column(String(200), default="")
```

Comment:
shouldn't be `github` login also unique?

Reply:
Could be. ;-)

Reply:
I've made some amendments here.